src/api.js

```diff
@@ -0,0 +1,21 @@
+import DefaultAxios from 'axios';
+
+function getCsrfToken() {
+  const csrfTokenMeta = document.querySelector('meta[name="csrf-token"]');
```

Comment:
Is this required?

Reply:
We do it in retail and I copied this over from there, but I don't see any csrf stuff in the ajax setup so I think we can remove it.

Reply:
I don't think we want csrf because we're going cross domain which is incompatible, right?